### ISSUE TITLE
Feature Request: Pinterest block

### ISSUE THREAD

**Issue by reginabally** [wordpress-mobile/gutenberg-mobile#4418]
> I love everything, but I miss Spotify and Pinterest blocks on the app, I need to add those on desktop.

Requested in a [4-star app review](https://play.google.com/console/u/0/developers/7957760354032996428/app/4973361825681449040/user-feedback/review-details?reviewId=gp%3AAOqpTOEtujEwciwx7rhw-RDvCac9Lvhrqe0iov-WMDPmoT58x8X9GDzkH_E1jNiWvesG-bqDWkcbttG4xd4U4Q&corpus=PUBLIC_REVIEWS).

**Comment by fluiddot** [comment]
The Pinterest block is a variant of the Embed block and is actually available but not included in the block picker. It can be added by either:
1. Via the slash inserter by typing `/pinterest`.
2. Adding an Embed block and setting a Pinterest URL.

However, I'd like to note that the inline preview is not available yet, which could be tackled in the short term.

**Comment by reginabally** [comment]
Thanks, @fluiddot! I've updated the app review reply.

**Comment by mchowning** [comment]
I'll leave this open as a request for inline preview of the pinterest embed. Any opposition to me updating the issue title to reflect that @reginabally ?

**Comment by fluiddot** [comment]
I'd like to note that we already have an open issue ([reference](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3981)) related to enabling the inline preview for other providers, so I referenced this issue there too.

**Comment by mchowning** [comment]
Thanks @fluiddot ! With that link, do you think we should instead just close this and [the spotify support issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/4417), and instead route all these kinds of requests through the "more providers" issue you linked?

**Comment by fluiddot** [comment]
> Thanks @fluiddot ! With that link, do you think we should instead just close this and [the spotify support issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/4417), and instead route all these kinds of requests through the "more providers" issue you linked?

Yeah, we could use the issue https://github.com/wordpress-mobile/gutenberg-mobile/issues/3981 to group requests in relation to inline previews of other providers. In any case, I'd highlight that these referenced blocks (Spotify and Pinterest) were requested by users.
